frontend/src/pages/Filter.js

```diff
@@ -0,0 +1,128 @@
+import Page from "../components/Page";
+import Navigation from "../components/Navigation";
+import {useAuth} from "../auth/AuthProvider";
+import DocumentGallery from "../components/DocumentGallery";
+import {useEffect, useState} from "react";
+import {getSomeUserDocs} from "../services/api-service";
+import FilterForm from "../components/FilterForm";
+import BackButton from "../components/BackButton";
+import {useHistory} from "react-router-dom";
+
+export default function Filter () {
+
+    const {token, user} = useAuth()
+    const {error, setError} = useState()
```

Comment:
```suggestion
    const [error, setError] = useState()
```

die klammern sollten eckig sein denke ich